Formula/mercury.rb

```diff
@@ -1,9 +1,8 @@
 class Mercury < Formula
   desc "Logic/functional programming language"
   homepage "https://mercurylang.org/"
-  url "https://dl.mercurylang.org/release/mercury-srcdist-14.01.1.tar.gz"
-  sha256 "98f7cbde7a7425365400feef3e69f1d6a848b25dc56ba959050523d546c4e88b"
-  revision 1
+  url "http://dl.mercurylang.org/release/mercury-srcdist-20.01.1.tar.gz"
```

Comment:
Nitpick: can we use HTTPS?

Reply:
That's not a nitpick, there's no reason not to use HTTPS. Good catch.